kubernetes/samples/charts/util/setupLoadBalancer.sh

```diff
@@ -216,7 +245,8 @@ function createNginx() {
 
   if [ "$(helm list --namespace $NNAME | grep $NNAME |  wc -l)" = 0 ]; then
```

Comment:
This should be:
if [ "$(helm list --namespace $NSPACE| grep $NNAME |  wc -l)" = 0 ]; then

Reply:
done